server/src/main/java/org/spine3/server/event/enrich/ReferenceValidator.java

```diff
@@ -151,11 +157,52 @@ private static IllegalStateException noFieldException(
         throw new IllegalStateException(msg);
     }
 
-    private static IllegalStateException noFunction(Class<?> sourceFieldClass, Class<?> targetFieldClass) {
-        final String msg = format(
-                "There is no enrichment function for translating %s to %s",
-                sourceFieldClass,
-                targetFieldClass);
-        throw new IllegalStateException(msg);
+    private static void logNoFunction(Class<?> sourceFieldClass, Class<?> targetFieldClass) {
+        // Using `DEBUG` level to avoid polluting the `stderr`.
+        if (log().isDebugEnabled()) {
+            log().debug("There is no enrichment function for translating {} into {}",
+                        sourceFieldClass, targetFieldClass);
+        }
+    }
+
+    /**
+     * A wrapper DTO for the validation result.
```

Comment:
Renamed. Though the original idea was to avoid self-repeating: `ReferenceValidator.Result`.

I see your point, since many users will `static import` this class. And the magic is gone.